pcsx2/MemoryCardFile.cpp

```diff
@@ -45,6 +46,7 @@ static constexpr int MC2_ERASE_SIZE = 528 * 16;
 static const char* s_folder_mem_card_id_file = "_pcsx2_superblock";
 
 bool FileMcd_Open = false;
+u32 current_crc = 0;
```

Comment:
If this doesn't need to have global visibility, it should be static. Probably the same with the bool above.